### PR TITLE
Custom bash script

### PR DESCRIPTION
**Pull request by sushant12**

### CODE REVIEW

File: lib/handin/accounts.ex [hunk 1]
```diff
@@ -360,4 +360,40 @@ defmodule Handin.Accounts do
...
   end
+
+  def list_users() do
+    Repo.all(User)
+  end
+
+  def list_users(params) do
+    case Flop.validate_and_run(User, params, for: User) do
+      {:ok, {users, meta}} ->
+        %{users: users |> Enum.map(&Repo.preload(&1, :university)), meta: meta}
```
Comment (sushant12) [inline lib/handin/accounts.ex:371]: eta `users |> Repo.preload(:university)` garna mildaina?

File: lib/handin/accounts.ex [hunk 2]
```diff
@@ -360,4 +360,40 @@ defmodule Handin.Accounts do
...
+    |> Repo.preload(:university)
+    |> User.edit_changeset(attrs)
+  end
+
+  def update_user(user, attrs) do
+    case user
+         |> edit_changeset(attrs)
+         |> Repo.update() do
+      {:ok, user} ->
+        {:ok, Repo.preload(user, :university, force: true)}
```
Comment (sushant12) [inline lib/handin/accounts.ex:389]: eta `force: true` garna parcha ra?

Comment (padmaJS) [reply lib/handin/accounts.ex:389]: force true nagarda xa ni university_id of another university is mapped but the association remains the old one for some reason

File: lib/handin/accounts.ex (hunk 1 above)
Comment (padmaJS) [reply lib/handin/accounts.ex:371]: my bad milne raixa

**Pull request merged**